vignettes/howto-dm-rows.Rmd

```diff
@@ -48,26 +51,26 @@ With the changes confirmed, you execute the method again, this time with the arg
 Note that `in_place = FALSE` is the default: you must opt in to actually change data on the database.
 
 Each method has its own requirements in order to maintain database consistency.
-These involve constraints on primary key values as they are how rows are identified.
+These involve constraints on primary key values that uniquely identify rows.
 
 | Method | Requirements |
 |--------|--------------|
-| `dm_rows_insert()` | The primary keys must differ from existing records.|
+| `dm_rows_insert()` | Primary keys must differ from existing records.|
 | `dm_rows_update()` | Primary keys must match for all records to be updated.|
 | `dm_rows_patch()` | Updates missing values in existing records. Primary keys must match for all records to be patched.|
 | `dm_rows_upsert()` | Updates existing records and adds new records, based on the primary key.|
-| `dm_rows_delete()` | Removes matching records based on the primary key.|
+| `dm_rows_delete()` | Removes matching records based on the primary key. Primary keys must match for all records to be deleted.|
 | `dm_rows_truncate()` | Removes all records, only for tables in the changeset dm.|
```

Comment:
This is about to disappear.

Reply:
So should I already remove it from the documentation? Or we should keep it until it is actually removed?

Reply:
No need to keep it in the docs, #1031 is scheduled very soon.

Reply:
Okay, remove mentions of this function.